frontend/src/resources/stateLayers.js

```diff
@@ -1,118 +1,69 @@
+  { sourceLayer: "minnesota-1r6mxv", sourceURL: "mapbox://meghabyte.7gxkg2l1", id: "27" },
+  { sourceLayer: "mississippi-3nl299", sourceURL: "mapbox://meghabyte.18k9f9v1", id: "28" },
+  { sourceLayer: "missouri-al6asq", sourceURL: "mapbox://meghabyte.bsh422ex", id: "29" },
+  { sourceLayer: "montana-ajp2xu", sourceURL: "mapbox://meghabyte.4ztugbfj", id: "30" },
+  { sourceLayer: "nebraska-ctr88k", sourceURL: "mapbox://meghabyte.auhhg1qr", id: "31" },
+  { sourceLayer: "nevada-deuc62", sourceURL: "mapbox://meghabyte.9s8ancwz", id: "32" },
+  { sourceLayer: "new_hampshire-dnh8wl", sourceURL: "mapbox://meghabyte.abdwrkjp", id: "33" },
+  { sourceLayer: "new_jersey-bfsqx6", sourceURL: "mapbox://meghabyte.c6qg6wi7", id: "34" },
+  { sourceLayer: "new_mexico-3ikl2a", sourceURL: "mapbox://meghabyte.38lde3fu", id: "35" },
+  { sourceLayer: "new_york-d90nlv", sourceURL: "mapbox://meghabyte.63txha6r", id: "36" },
+  { sourceLayer: "north_carolina-7lsq0k", sourceURL: "mapbox://meghabyte.414b3sc4", id: "37" },
+  { sourceLayer: "north_dakota-8ndoqs", sourceURL: "mapbox://meghabyte.91tvka7l", id: "38" },
+  { sourceLayer: "ohio-6qh2hg", sourceURL: "mapbox://meghabyte.cph75mnf", id: "39" },
+  { sourceLayer: "oklahoma-43tc4i", sourceURL: "mapbox://meghabyte.80ydutmo", id: "40" },
+  { sourceLayer: "oregon-b400hj", sourceURL: "mapbox://meghabyte.74vanshz", id: "41" },
+  { sourceLayer: "pennsylvania-32hw1n", sourceURL: "mapbox://meghabyte.74giprnb", id: "42" },
+  { sourceLayer: "puerto_rico-c4f903", sourceURL: "mapbox://meghabyte.6ga7s4y8", id: "72" },
+  { sourceLayer: "rhode_island-4o9n9e", sourceURL: "mapbox://meghabyte.8d1m392l", id: "44" },
+  { sourceLayer: "samoa-0upg6q", sourceURL: "mapbox://meghabyte.dsgyamz0", id: "samoa" },
+  { sourceLayer: "south_carolina-1fp1wa", sourceURL: "mapbox://meghabyte.94zwzxhh", id: "45" },
+  { sourceLayer: "south_dakota-3mlxen", sourceURL: "mapbox://meghabyte.cnh2yp9s", id: "46" },
+  { sourceLayer: "tennessee-5qqvkf", sourceURL: "mapbox://meghabyte.47ibvcmi", id: "47" },
+  { sourceLayer: "texas-5xquxl", sourceURL: "mapbox://meghabyte.4296m2og", id: "48" },
+  { sourceLayer: "utah-2kvqon", sourceURL: "mapbox://meghabyte.ay7zbcto", id: "49" },
+  { sourceLayer: "vermont-b2t4kt", sourceURL: "mapbox://meghabyte.ckvqs682", id: "50" },
+  { sourceLayer: "virgin_islands-0zjitm", sourceURL: "mapbox://meghabyte.7jxhqi3g", id: "78" },
+  { sourceLayer: "virginia-36qhi0", sourceURL: "mapbox://meghabyte.c2t8jtng", id: "51" },
+  { sourceLayer: "washington-76b60f", sourceURL: "mapbox://meghabyte.8f91q04h", id: "53" },
+  { sourceLayer: "west_virginia-8jz02z", sourceURL: "mapbox://meghabyte.4lueicdj", id: "54" },
+  { sourceLayer: "wisconsin-32yyxh", sourceURL: "mapbox://meghabyte.3iwzwwxv", id: "55" },
+  { sourceLayer: "wyoming-758cbr", sourceURL: "mapbox://meghabyte.2zqpgoup", id: "56" },
 ];
 
 export let sourceURLs = [];
 
 stateLayers.map(stateLayer => {
   sourceURLs.push(stateLayer.sourceURL);
 });
```

Comment:
do we still need this at all after your change?

Reply:
prob not